src/bin/cmd/wallet.rs

```diff
@@ -192,7 +195,7 @@ pub fn wallet_command(wallet_args: &ArgMatches, config: GlobalWalletConfig) {
 					.expect("Failed to parse number of change outputs.");
 				let fluff = send_args.is_present("fluff");
 				let max_outputs = 500;
-				if dest.starts_with("http") {
+				if method == "http" {
```

Comment:
Perhaps output an explicit error if the destination provided doesn't look like a URL?